src/ui/constants/colors.ts

```diff
@@ -0,0 +1,6 @@
+export const COLORS = {
+    lightBlue: '#93E3FE',
+    paleGreen: '#B0DC8B',
+    paleYellow: '#FDF894',
```

Comment:
just a naming nitpick, but why not stick with either "pale" or "light"?

Reply:
😂 I'm usually bad at choosing names, so I followed chatGPT suggestions for the color names, but you're right, let's stick to 'light'🤣